src/components/Search/SearchList.tsx

```diff
@@ -295,8 +304,77 @@ function SearchList(
 
     useImperativeHandle(ref, () => ({scrollAndHighlightItem, scrollToIndex}), [scrollAndHighlightItem, scrollToIndex]);
 
+    const getItemHeight = useCallback(
+        (item: SearchListItem): number => {
+            try {
+                const reportListItem = item as ReportListItemType;
+                const transactionListItem = item as TransactionListItemType;
+                const reportActionListItem = item as ReportActionListItemType;
```

Comment:
We also have a task list item

Reply:
The task list is handled by this part and it has default sizes ```return isLargeScreenWidth ? variables.searchListItemHeightLargeScreen : variables.searchListItemHeightSmallScreen;```

Reply:
The task list is handled by this part and it has default sizes return isLargeScreenWidth ? variables.searchListItemHeightLargeScreen : variables.searchListItemHeightSmallScreen;